install_sct

```diff
@@ -757,5 +756,4 @@ fi
 
 ) 2>&1 | tee install_sct_log.txt
 status=${PIPESTATUS[0]}
-popd >/dev/null
```

Comment:
While you're at it, you could inline the definition of the `status` variable into the `exit` command, since there's no longer a `popd` in between to clobber `PIPESTATUS`.